lib/senkyoshi/models/wikipage.rb

```diff
@@ -43,12 +44,21 @@ def _set_body(original_body, url, extendeddata)
           #{body}
         }
       end
+      if @referred_to_title.present?
+        body = %{
+          <a href="$CANVAS_COURSE_REFERENCE$#{@referred_to_title}">
+            Course Link: #{@referred_to_title}
```

Comment:
I added this "Course Link: " label before the title to provide more context to users as to what it's supposed to be linking to. If you think it'd be better to leave this off or change it to something else, let me know.